examples/wasmfleetautoscaler.yaml

```
@@ -0,0 +1,51 @@
+---
+# Copyright 2018 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Full example of a FleetAutoscaler that is using wasm for it's custom
+# aautomation logic - this is used to scale a Fleet.
+#
+
+#
+# For a full reference and details: https://agones.dev/site/docs/reference/fleetautoscaler/
+#
+apiVersion: autoscaling.agones.dev/v1
+kind: FleetAutoscaler
+metadata:
+  name: webhook-fleet-autoscaler
+spec:
+  fleetName: simple-game-server
+  policy:
+    # type of the policy - this example is Webhook
+    type: Wasm
+    # parameters for the wasm policy
+    wasm:
+      # The exported function to call in the wasm module, defaults to 'scale'
+      function: 'scale'
+      # Config values to pass to the wasm program on startup
+      config:
+        scale_buffer: 10
```

It might be `buffer_size` instead, following your main.go 🙂 

```suggestion
        buffer_size: 10
```